packages/docusaurus/src/server/routes.ts

```diff
@@ -272,7 +272,9 @@ ${JSON.stringify(routeConfig)}`,
   res.routesChunkNames[`${routePath}-${routeHash}`] = {
     // Avoid clash with a prop called "component"
     ...genChunkNames({__comp: component}, 'component', component, res),
-    ...genChunkNames({__context: context}, 'context', routePath, res),
+    ...(context
+      ? genChunkNames({__context: context}, 'context', routePath, res)
+      : {}),
```

Comment:
not important but you could replace ternary by && here

<img width="108" alt="image" src="https://user-images.githubusercontent.com/749374/162253663-5f66d14d-1268-4538-846d-87358f5fa98f.png">